mcu/stm32f4xx.S

```diff
@@ -188,6 +187,22 @@ Reset_Handler:
     cbz     r4, .L_start_boot
 #endif
 
+#if ( (defined(DFU_VERIFY_CHECKSUM) && (DFU_VERIFY_CHECKSUM != _DISABLE) ) )
+
+    mov     r0, #0xFFFFFFFF
+    ldr     r1, =#_APP_START
+    ldr     r2, =__romend
+    sub     r2, r2, r1
+    sub     r2, #8
```

Comment:
As per conversation, the adjustment should use a define from the checksum routine to determine the length, this would allow the checksum to be replaced with SHA or some other more advanced routine.

Reply:
Fixed https://github.com/dmitrystu/sboot_stm32/pull/10/commits/0b643c2f37857561fb61ae8bd86335b32ecb72a4